### DISCUSSION TITLE
Changer des sous-catégories dans "Secteurs d'activité"?

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1778] [Retours utilisateur·rice]
Une utilisatrice m'a fait remarquer que dans le champ "Secteurs d'activité" du formulaire de création de cantine, il y a d'après elle trop de sous-catégories dans la catégorie "Administration". Plutôt que d'avoir 6 sous-catégories, elle se contenterait de deux : "RA" et "RIA".

Par ailleurs, dans la catégorie "Enseignement", la formulation "Secondaire collège" a induit en erreur un utilisateur : il pensait que ça incluait et la partie collège et la partie lycée de sa cantine. Cela m'a fait penser que, par souci de simplicité, on pourrait enlever le mot "Secondaire" dans chacune des sous-catégories suivantes : "Secondaire Lycée agricole", "Secondaire collège" et "Secondaire lycée (hors agricole)". On pourrait aussi inverser la place de "Secondaire Lycée agricole" et "Secondaire collège" par souci de classement chronologique.

Votre avis, @JenniferStephan et @gauthierandre?

**Comment by alemangui** [comment]
@valeriemerle t'en penses quoi ?

**Reply by valeriemerle** [reply]
oui j suis d'accord que la sectorisation est trop complexe et induit pas mal d'erreur d'aiguillage dans les saisies utilisateurs et n'est pas optimales dans l'affichage des stat. Est ce possible cependant de arder les différents sous secteurs mais faire un affichage par grands secteurs (comme sur le tableau de bord du site) et affiner par ascenseur? et pas une liste de tous les secteurs à la fois sans ordre de classement, 
je vais proposer une liste.
Est ce possible  au passage de mettre en français les noms de tous les secteurs dans le tableau bord ma cantine? Merci

**Reply by hfroot** [reply]
Quels secteurs sont en anglais, et ou précisement (par ex un URL) ?
